src/GitReleaseManager.Core/Configuration/ConfigurationProvider.cs

```diff
@@ -99,7 +99,19 @@ public static void WriteSample(string targetDirectory, IFileSystem fileSystem)
 
         private static string GetConfigFilePath(string targetDirectory)
         {
-            return Path.Combine(targetDirectory, "GitReleaseManager.yaml");
+            var filePath = Path.Combine(targetDirectory, "GitReleaseManager.yaml");
+            if (File.Exists(filePath))
+            {
+                return filePath;
+            }
+
+            filePath = Path.Combine(targetDirectory, "GitReleaseManager.yml");
+            if (File.Exists(filePath))
```

Comment:
There shouldn't be a need for the check here IMO.
If `GitReleaseManager.yaml` isn't found, then we should always return `GitReleaseManager.yml` from this method.

Reply:
I had the same thought, but then this would mean running File.Exists again on the returned value, meaning that we could have actually tested the same file twice.  I thought it better to only test the path once, and then knowingly return a null when neither of them can be found.

Reply:
fair enough